### ISSUE TITLE
heatmap visibility in volumeslices can't be toggled independently

### ISSUE THREAD

**Issue by tomchor** [MakieOrg/Makie.jl#2873]
I'm experiencing what might be a bug in the [volumeslices example](https://docs.makie.org/stable/examples/plotting_functions/volumeslices/) in the docs.

After I run everything the plot appears as it should, but toggling any of the plots using the switches on the RHS, all of them disappear.

Is this a bug?

**Comment by SimonDanisch** [comment]
Yes, this is a bug... I guess they all share the same visible observable from the recipe.

**Comment by ffreyer** [comment]
Should just require something like `heatmap(..., visible = map(identity, visible))` with `visible = pop!(attr, :visible)`.

**Comment by ffreyer** [comment]
This was fixed at some point
